source/ranges.tex

```diff
@@ -6146,7 +6144,7 @@
 
   private:
     template<bool> struct @\exposid{iterator}@;                     // \expos
-    template<bool> struct @\exposid{sentinel}@;                     // \expos
+    template<bool> struct @\exposid{sentinel}@;                     // \expos{} (see \ref{range.elements.sentinel})
```

Comment:
@CaseyCarter's review shows the correct way to cross-reference.

Reply:
We're (sadly) inconsistent in how we add references for non methods and one-liners, but I see that we've used the suggested referencing for \exposid{sentinel}s elsewhere, so I'll follow that convention here.